apps/dashboard/README.md

```diff
@@ -38,7 +38,8 @@ themselves.
 It should be noted here that any Ruby module installation needs configured with
 `--enable-shared` flag to be compatible with the Ruby running on the webnode.
 
-To build run `bin/setup` from within this directory.
+Now build run `bin/setup` from within this directory to fetch all the dependencies
```

Comment:
```suggestion
Now build by running `bin/setup` from within this directory to fetch all the dependencies
```

Reply:
```suggestion
Now run `bin/setup` from within this directory to fetch all the dependencies
```

Reply:
2 comments, same thing. Sorry I'm going to take Travis'.